README.md

```diff
@@ -1,75 +1,89 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
-
-> Description the project.
+# To-do List
 
+> In this project, I will build a simple HTML list of To Do tasks. "To-do list" is a tool that helps to organize your day. It simply lists the things that you need to do and allows you to mark them as complete. This web page will be built using webpack and served by a webpack dev server.
 
 ## Built With
 
-- Major languages
-- Frameworks
-- Technologies used
-
-## Live Demo (if available)
+- HTML and CSS
+- Frameworks: N/A
+- Technologies used: GIT, GITHUB, LINTERS
+- npm
+- Webpack
 
-[Live Demo Link](https://livedemo.com)
+## Live Demo
 
+Check **Live Demo** [here.](https://sonickmumba.github.io/To-do-list/)
```

Comment:
[OPTIONAL] This live demo link is broken. It would be preferable to fix this. Your README is the entry point of your users and having it well written without any broken links gives a good first impression and maximizes the potential of your product to your users.